aboutMe/apps.js

```diff
@@ -0,0 +1,55 @@
+//Q1
+const page=document.querySelector("page");
```

Comment:
You should select body
```suggestion
const page=document.querySelector("body");
```